srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -852,6 +852,9 @@ def main():
     qclogging.add_general_file_handler(primary_logger, "rel2srf.txt")
     args = load_args()
     realisation = load_realisation_file_as_dict(args.realisation_file)
+    realisation["name"] = str(
```

Comment:
Might be good to add this to the load_realisation_file_as_dict function? Is kinda part of the loading step and prevents any further issues (i.e. other places that might use the load_realisation_file_as_dict function).

Reply:
I agree with Claudio on this one